.vib/odoo/cypress/cypress/integration/odoo_spec.js

```diff
@@ -10,6 +10,13 @@ it('allows installing/uninstalling an application and inviting new users', () =>
   });
   cy.reload();
 
+  // Perform the second login only if the #login selector is visible
+  cy.get('#login').should('be.visible').then($login => {
```

Comment:
This logic will fail if, for whatever reason, the selector `#login` is not present. Check how this is done for kibana or mediawiki for example, you need to get the page body and then check if a particular selector is present.